security/sanitizers.rst

```diff
@@ -0,0 +1,352 @@
+.. _security_sanitizers:
+
+Sanitizers
```

Comment:
For discoverability, search purposes, something like ``C, C++ compiler sanitizers`` or something in that line, a bit more complete, could be nice